package.sh

```diff
@@ -10,8 +10,17 @@ init_ve() {
 }
 
 package() {
-  sudo rm -r dist
-  pyinstaller --clean --hidden-import=requests main.py
+  local arch=$(uname -m)
+  if [ "$arch" == "x86_64" ]; then
+    if [ -d dist/main ]; then
+      sudo rm -r dist/main
+    fi
+    pyinstaller --clean --hidden-import=requests main.py
```

Comment:
also, i think we should just run the pyinstaller only once. this should be the logic here
- check arch
- build the binary in the respective folder `/dist/arch/main//`
- if its x86, copy the same binary from the `/dist/arch/main/..` folder to `dist` folder
- even better would be if we can just symlink `dist` folder to `/dist/arch/main`.
- basically we want to add minimum code to achieve this. lets verify first what works